app/api/controllers/requestParsers/validators/validations/DateValidation.scala

```diff
@@ -37,11 +38,11 @@ object DateValidation {
       case _ => Nil
     }
 
-  def validate(field: String, isFromDate: Boolean, minYear: Int, maxYear: Int): List[MtdError] = {
+  def validate(field: String, isFromDate: Boolean): List[MtdError] = {
```

Comment:
One issue here is that the DateValidation is no longer as general-purpose without the min and maxYear args. Perhaps could keep them, but add default values to each arg, i.e.
```
, minYear: Int = defaultMinYear, ...
```
(where `val defaultMinYear = 1900` )